addons/web/static/src/legacy/scss/ui.scss

```diff
@@ -38,6 +38,11 @@
     }
 }
 
+.o_kbd {
```

Comment:
Doing so you'll have to both use a `<kbd>` tag and apply the class to get the desired result.
Just target  `kbd {}` tag in scss ;)

Also, move this in webclient.scss

Reply:
@stefanorigano 👋 
It's done 👍